modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/impl/IngestServiceImpl.java

```diff
@@ -266,11 +267,11 @@
   private Cache<String, Long> partialTrackStartTimes = CacheBuilder.newBuilder().expireAfterWrite(1, TimeUnit.DAYS)
           .build();
 
-  /** The default is to overwrite series catalog on ingest */
-  protected boolean defaultIsOverWriteSeries = true;
+  /** The OC default is to overwrite existing flavors on ingest. The default is used if the configuration param is not found */
+  protected boolean defaultOverWriteValue = true;
```

Comment:
I'm wondering if `false` wouldn't be a better default since my guess would be that it is more common for users to modify data on the admin interface than on the capture agents since most capture agents (SMP, NCast, pyCA, ARec) by default do not even give you the option to modify this data.

Reply:
And you could make this `final` since it's basically a constant isn't it?

Reply:
Using 'false' as default would be good for us. However, currently, in OC v2x-v7x, ingesting catalogs overwrite the existing catalogs. The 'true' default, and the current 'true' behavior, works well for Galicaster capture agents. Galicaster  has the use case that event titles, and other event metadata, are adjusted on the CA UI, in the classroom, at the point of capture. I am concerned that flipping the default behavior from true to false will cause metadata data loss to Galicaster adopters until they adjust their config.

Reply:
I vote for keeping the existing behaviour as the default as this match the flag for series overriding. If we change the default then both should be changed at the same time.

Reply:
@JamesUoM I realize that overwrite value will change from default true to default false pretty soon. Not sure if it's worth changing now or wait for the merge back from @lkiesow 's other overwrite pull.

Reply:
I made the default constants final